app/src/main/java/com/junkfood/seal/MainActivity.kt

```diff
@@ -73,6 +88,36 @@ class MainActivity : AppCompatActivity() {
         handleShareIntent(intent)
     }
 
+    fun requestExternalStoragePermission(activity: Activity) {
+        // Check if the app has the necessary permissions
+        if (ContextCompat.checkSelfPermission(
+                activity,
+                android.Manifest.permission.WRITE_EXTERNAL_STORAGE
```

Comment:
What's the point of `WRITE_EXTERNAL_STORAGE` and `READ_EXTERNAL_STORAGE` here? These two are deprecated in API 30

Reply:
I'm not sure if they are needed for saving files in the SD in devices with less API level.

Reply:
Direct access to SD Card is not possible since API 21 afaik, these two permissions is requested [here](https://github.com/JunkFood02/Seal/blob/main/app/src/main/java/com/junkfood/seal/ui/page/download/DownloadPage.kt#L124) already

Reply:
Yes, I saw that. Sorry for it!